cmake/PrintHelper.cmake

```diff
@@ -59,7 +59,6 @@ function(print_configuration)
   endif()
 
   print_variables( VARS
-    "BUILD_SHARED_LIBS;Build shared libraries"
```

Comment:
Is there an easy way to inform people that still use `-DBUILD_SHARED_LIBS=ON` that this is not needed anymore?

Reply:
The option defaulted to building shared since bb83edc7a638947ccb476320dcdb8473c9098e2e or release 2.3.0 of preCICE.

There is no reason to explicitly set this ever since. I think this shouldn't really be a problem.

Reply:
Not a problem, I was just thinking of an effective way to inform people to clean up their scripts. It's already ok as-is.